### PR TITLE
Cache images from active storage

### PR DESCRIPTION
**Pull request by hellcp-work**
Making sure that files provided by active storage are cached

### CODE REVIEW

**Comment by eduardoj** [comment]
We could maybe make use of the `config.active_storage.urls_expire_in` setting... See: https://blog.saeloun.com/2021/09/14/rails-7-adds-expiring-urls-to-active-storage/

File: src/api/Procfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 web: bin/rails server -b 0.0.0.0
 delayed: bundle exec script/delayed_job.api.rb run
 clock: bin/clockworkd --log-dir=log -l -c config/clock.rb run
-search: bin/rails sphinx:start_for_development
+#search: bin/rails sphinx:start_for_development
```
Comment (eduardoj) [inline src/api/Procfile:4]:
```suggestion
search: bin/rails sphinx:start_for_development
```

This looks like a leftover for making tests...

Comment (hellcp-work) [reply src/api/Procfile:4]: oops, indeed

**Review by hennevogel — APPROVED** [review]

**Comment by eduardoj** [comment]
> We could maybe make use of the `config.active_storage.urls_expire_in` setting... See: https://blog.saeloun.com/2021/09/14/rails-7-adds-expiring-urls-to-active-storage/

I tested it. It doesn't work as expected. :thinking:

**Review by eduardoj — APPROVED** [review]

**Pull request merged**